website/docs/docs/use-dbt-semantic-layer/gsheets.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Google Sheets (beta)"
+description: "Integrate with Google Sheets to query your metrics in a spreadsheet."
+tags: [Semantic Layer]
+sidebar_label: "Google Sheets (beta)"
+---
+
+:::info Beta functionality
+Google Sheets integration with the dbt Semantic Layer is a [beta feature](https://docs.getdbt.com/docs/dbt-versions/product-lifecycles#dbt-cloud) and is subject to change without notification.
+:::
+The dbt Semantic Layer offers a seamless integration with Google Sheets through a custom menu. This add-on allows you to build dbt Semantic Layer queries and return data on your metrics directly within Google Sheet.
+
+## Prerequisites
+
+1. You have a Google account with access to Google Sheets
+2. You have the ability to install Google Add-ons 
+3. You have [set up the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl)
+4. You have a dbt Cloud Environment Id and a [service token](/docs/dbt-cloud-apis/service-tokens) to authenticate with from a dbt Cloud account 
+
+## Installing the add-on 
+
+1. In Google Sheets, navigate to [**Extensions -> Add-on -> Get add-ons**](https://support.google.com/docs/answer/2942256?hl=en&co=GENIE.Platform%3DDesktop&oco=0#zippy=%2Cinstall-add-ons%2Cinstall-an-add-on).
+2. Search for "dbt Semantic Layer for Sheets" and install it
+3. After installing, access it by opening the Add-On menu and finding "dbt Semantic Layer for Sheets". This will open a custom menu to the right hand side of your screen.
```

Comment:
I'm not seeing these options in our Google Sheets environment. Are there specific permissions required for this that we should call out in the prerequisites? Have you seen this in action and if so, is it something you think will be easy to find? Should we include screenshots?
```suggestion
3. After installing, open the Add-On menu and select the "dbt Semantic Layer for Sheets". This will open a custom menu to the right-hand side of your screen.
```